superset-frontend/src/views/components/SubMenu.tsx

```diff
@@ -231,6 +243,22 @@ const SubMenuComponent: React.FunctionComponent<SubMenuProps> = props => {
           })}
         </Menu>
         <div className={navRightStyle}>
+          <Menu mode="horizontal">
+            {props.dropDownLinks?.map((link, i) => (
+              <SubMenu key={i} title={link.label} icon={<Icons.TriangleDown />}>
+                {link.childs?.map(item => {
+                  if (typeof item === 'object') {
```

Comment:
do you need this check since all values in the `dropDownLinks` array is an object?

Reply:
This check is for typescript. It was complaining since child type could be string.